### PR TITLE
build-configs.yaml: remove lee_android_3.18 build config

### PR DESCRIPTION
**Pull request by nuclearcat**
Remove lee_android_3.18 as requested in https://github.com/kernelci/kernelci-core/issues/235#issuecomment-1122124032

Signed-off-by: Denys Fedoryshchenko <denys.f@collabora.com>

### CODE REVIEW

**Comment by gctucker** [comment]
This is not the same issue - please see my comments.  The lee_android_3.18 build config is not an Android tree.

**Comment by gctucker** [comment]
> This is not the same issue - please see my comments. The lee_android_3.18 build config is not an Android tree.

As confirmed by Lee in an other comment, this can be dropped.

File: config/core/build-configs.yaml [hunk 1]
```diff
@@ -706,11 +706,6 @@ build_configs:
     <<: *kselftest-tree
     branch: 'next'
 
-  lee_android_3.18:
-    tree: lee
```
Comment (gctucker) [inline config/core/build-configs.yaml:710]: This branch was the only one using the `lee` tree, so we might also drop it.

@lag-linaro Are there other branches in your tree you would like to have covered by KernelCI?

Comment (lag-linaro) [reply config/core/build-configs.yaml:710]: No, not right now, thank you.

Comment (nuclearcat) [reply config/core/build-configs.yaml:710]: I will update PR accordingly.

**Review by gctucker — APPROVED** [review]

**Pull request merged**